dev/gpctl/cmd/api/workspaces-get.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package api
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	v1 "github.com/gitpod-io/gitpod/public-api/v1"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+func newWorkspacesGetCommand() *cobra.Command {
+	var workspaceID string
+
+	cmd := &cobra.Command{
+		Use:     "get",
+		Short:   "Retrieve details about a workspace by ID",
+		Example: "  get --id 1234",
```

Comment:
Here we use `--id`, in the owner token we use `--workspace-id`, we should be consistent across these 2.

Reply:
fixed